innovation_sweet_spots/analysis/notebooks/foodtech/research_funding/research_step3_analysis.py

```diff
@@ -470,36 +492,33 @@
 category_amount_magnitude_growth
 
 # %%
+# In domain list, health is last item in the list so that
+# if domain_excl_health is used instead of domain
+# the colours will remain the same.
 domain = [
-    "Health",
     "Innovative food",
     "Logistics",
     "Restaurants and retail",
     "Cooking and kitchen",
     "Food waste",
+    "Health",
 ]
 range_ = pu.NESTA_COLOURS[0 : len(domain)]
 
-# %%
-from innovation_sweet_spots.utils import chart_trends
-import importlib
-
-importlib.reload(chart_trends)
-
-# %%
-category_amount_magnitude_growth.magnitude.median()
+domain_excl_health = domain[:-1]
+range_excl_health = pu.NESTA_COLOURS[0 : len(domain_excl_health)]
```

Comment:
By moving "Health" here to the last item in the list (rather than first), it means the colours will remain the same when plotting charts with and without the "Health" category